src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -0,0 +1,148 @@
+import { Registration, UnpublishPublicationRequest } from '../../../types/registration.types';
+import { Box, Breadcrumbs, Button, DialogActions, Divider, TextField, Typography } from '@mui/material';
+import { useState } from 'react';
+import { Modal } from '../../../components/Modal';
+import { ErrorMessage, Field, FieldProps, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import { useTranslation } from 'react-i18next';
+import { FindRegistration } from './FindRegistration';
+import { unpublishRegistration } from '../../../api/registrationApi';
+import { setNotification } from '../../../redux/notificationSlice';
+import { useDispatch } from 'react-redux';
+import { LoadingButton } from '@mui/lab';
+import { RequiredDescription } from '../../../components/RequiredDescription';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useHistory } from 'react-router-dom';
+import i18n from '../../../translations/i18n';
+import { useMutation } from '@tanstack/react-query';
```

Comment:
NÅ mener jeg @joachimjorgensen har funnet en løsning på sortering av imports? :D

Reply:
Skrudde på en settings tror det er løsninga.

Reply:
det som skal skje er at alle eksterne imports kommer øverst, mens våre interne komponenter kommer nederst (som resultat av alfabetisering da)

Reply:
Fant ut at jeg hadde en setting for javascript og en annen for typescript. Håper det er på plass nå,